player.js

```diff
@@ -0,0 +1,58 @@
+Scrabble = require('./scrabble');
+var scrabble = new Scrabble();
+
+var Player = function(name){
+  this.plays = [];
+  this.name  = name;
+
+  this.play = function(word) {
+    if (this.hasWon === true) {
+      return false;
+    }
+    this.plays.push(word);
+    return this.plays;
+  };
```

Comment:
it seems messy to me to have a function that either returns an array, or false. I'd recommend reworking the method so it returns both, like ```return this.plays, false```. This makes it a more reliable function to call and use